### ISSUE TITLE
[Feedback Ticket] 508-defect-2 [SEMANTIC MARKUP]: Links SHOULD not have title attributes

### ISSUE THREAD

**Issue by productmike** [department-of-veterans-affairs/va.gov-cms#14244]
## Description
Continuation of #11843

## Acceptance Criteria

- [ ] Fix from #11843 posted to drupal community for feedback.
- [ ] Receive and evaluate feedback received with PM (Mike) and a11y SME (Laura). 
- [ ] Make updates as discussed with a11y SME and PM.
 

------Snippet of 11843 below-------

## Steps to Recreate

1. Go to the [Erie medical Center page](https://www.va.gov/erie-health-care/locations/erie-va-medical-center/)
2. Hover over "Learn more about DAV and the other van services available in your county" link
3. Verify that a small "tool tip" like bit of text appears

## Additional Context
- This title attribute is not being added manually by the editor, I confirmed in the CMS
**Screenshot of Link in the Rich text editor - there is no title:**
![image](https://user-images.githubusercontent.com/106678594/206198868-821d6265-b531-4717-bbdd-3340756d2978.png)

**Screenshot of the View node  - now there is a title:**
![image](https://user-images.githubusercontent.com/106678594/206199113-885594f3-fc6c-47f9-beaf-67c44faebdcd.png)

- It's also odd that every link on this node is not getting a title, none of these have it:
![image](https://user-images.githubusercontent.com/106678594/206199368-6764cc46-8268-4afd-bd77-ec10e81561a5.png)

- This is also not just happening on Facilities pages, I have an issue of this occurring very recently on a [Benefits page](https://prod.cms.va.gov/education/about-gi-bill-benefits/post-9-11):
![image](https://user-images.githubusercontent.com/106678594/206199542-b7b34f60-85fe-4dc6-b146-33c57abfdb99.png)

**Comment by productmike** [comment]
@edmund-dunn @tonytaylor @ndouglas 
Can y'all include this in your pre-finement discussions prior to next week's refinement session?

cc: @laflannery @BerniXiongA6

**Comment by srancour** [comment]
Still an issue and should be addressed.
